### PR TITLE
Docs: Improve CSP suggestion

### PR DESCRIPTION
**Pull request by mjansenDatabay**
This PR suggest an improvement for the CSP documentation of the `secure.md`.

### CODE REVIEW

File: docs/configuration/secure.md [hunk 1]
```diff
@@ -265,6 +265,26 @@ add_header X-Content-Security-Policy "default-src 'self'; connect-src 'self'; sc
 
 see also: [Browser compatibility of HTTP headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP#Browser_compatibility)
 
+The proposed CSP is to be understood as a starting point. ILIAS as a generic software to support LMS scenarios could
```
Comment (kergomard) [inline docs/configuration/secure.md]:
```suggestion
The proposed CSP is to be understood as a starting point. ILIAS is a generic software to support many different LMS scenarios and thus can't
```

File: docs/configuration/secure.md [hunk 2]
```diff
@@ -265,6 +265,26 @@ add_header X-Content-Security-Policy "default-src 'self'; connect-src 'self'; sc
...
+not provide a suggestion which fits all circumstances by guaranteeing the best security.
```
Comment (kergomard) [inline docs/configuration/secure.md]:
```suggestion
provide a suggestion which fits all circumstances and guarantees the best security.
```

File: docs/configuration/secure.md [hunk 3]
```diff
@@ -265,6 +265,26 @@ add_header X-Content-Security-Policy "default-src 'self'; connect-src 'self'; sc
...
+Depending on your (especially external) user content (liked embedded media, SCORM packages, etc.) you should try to
```
Comment (kergomard) [inline docs/configuration/secure.md]:
```suggestion
Depending on the content your users can provide (embedded media, SCORM packages, etc.) you should try to
```

File: docs/configuration/secure.md [hunk 4]
```diff
@@ -265,6 +265,26 @@ add_header X-Content-Security-Policy "default-src 'self'; connect-src 'self'; sc
...
+initially define a CSP which is as strict as possible.
+
+You could use the [Reporting Feature](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP#enabling_reporting) to
+log and analyse CSP violations to finally make the CSP (carefully) more loose, if necessary.
```
Comment (kergomard) [inline docs/configuration/secure.md]:
```suggestion
log and analyse CSP violations to loosen the CSP gradually and carefully, if necessary.
```

**Review by kergomard — COMMENTED** [review]
Thank you very much @mjansenDatabay !

I just propose a few grammatical changes to render the text, I think, easier to read.

Thanks again and best,
@kergomard

**Comment by nicoroeser** [comment]
This change introduces a good starting point for users, thanks!

**Pull request merged**